test/decorators/swagger/project_decorator_test.rb

```diff
@@ -26,5 +27,18 @@ class ProjectDecoratorTest < ActiveSupport::TestCase
       refute @decorator.context[:use_resource_representation_name_as_uid]
       refute @decorator.to_swagger.include? '#/components/schemas/DefaultPokemon'
     end
+
+    test 'generated json contains valid security schemes' do
```

Comment:
I could not check the produced json because generated data is not ISO across tests.